mParticle-Radar/MPKitRadar.m

```diff
@@ -103,28 +102,77 @@ - (MPKitExecStatus *)didBecomeActive {
     return [[MPKitExecStatus alloc] initWithSDKCode:[MPKitRadar kitCode] returnCode:MPKitReturnCodeSuccess];
 }
 
-#pragma mark Events
+#pragma mark User attributes and identities
+
+- (NSString*) getMpId:(FilteredMParticleUser *)user {
+    if (user != nil && user.userId != nil && user.userId.integerValue != 0) {
+        return [user.userId stringValue];
+    } else {
+        return nil;
+    }
+}
+
+- (void)setMparticleMetadata:(NSString *)mpId {
+    if ([Radar getMetadata] != NULL) {
+        NSMutableDictionary *userMetadata = [[Radar getMetadata] mutableCopy];
+        [userMetadata setObject:mpId forKey:@"mParticleId"];
+        NSDictionary *userMetadataDict = [userMetadata copy];
+        [Radar setMetadata:userMetadataDict];
+    }
+    else {
+        NSDictionary *userMetadataDict = @{@"mParticleId" : mpId};
```

Comment:
Nit: We generally do `@{@"mParticleId": mpId}` instead of `@{@"mParticleId" : mpId}`, but either is acceptable